metabase/metabase
# Coerce the default parameter value when the parameter is no longer multi-select

Fixes https://github.com/metabase/metabase/issues/44666

How to verify:
- Add a parameter to a dashboard and map it to Orders.Category
- Set the default value to "Gadget", "Widget"
- Set "Single value"
- The default parameter value should be changed to "Gadget"

<img width="1595" alt="Screenshot 2024-06-25 at 11 31 47" src="https://github.com/metabase/metabase/assets/8542534/b3ba3854-a223-4685-ad2e-4936c43d120c">
